skyportal/tests/frontend/test_followup_requests_LT.py

```diff
@@ -0,0 +1,132 @@
+import uuid
+import pytest
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.common.action_chains import ActionChains
+from skyportal.tests import api
+
+
+def add_telescope_and_instrument(instrument_name, token):
+    status, data = api("GET", f"instrument?name={instrument_name}", token=token)
+    if len(data["data"]) == 1:
+        return data["data"][0]
+
+    telescope_name = str(uuid.uuid4())
+    status, data = api(
+        "POST",
+        "telescope",
+        data={
+            "name": telescope_name,
+            "nickname": telescope_name,
+            "lat": 0.0,
+            "lon": 0.0,
+            "elevation": 0.0,
+            "diameter": 10.0,
+            "robotic": True,
+        },
+        token=token,
+    )
+    assert status == 200
+    assert data["status"] == "success"
+    telescope_id = data["data"]["id"]
+
+    status, data = api(
+        "POST",
+        "instrument",
+        data={
+            "name": instrument_name,
+            "type": "imager",
+            "band": "Optical",
+            "telescope_id": telescope_id,
+            "filters": ["ztfg"],
+            "api_classname": "%sAPI" % instrument_name,
+        },
+        token=token,
+    )
+    assert status == 200
+    assert data["status"] == "success"
+    return data["data"]
+
+
+def add_allocation(instrument_id, group_id, token):
+    status, data = api(
+        "POST",
+        "allocation",
+        data={
+            "group_id": group_id,
+            "instrument_id": instrument_id,
+            "hours_allocated": 100,
+            "pi": "Ed Hubble",
+        },
+        token=token,
+    )
+    assert status == 200
+    assert data["status"] == "success"
+    return data["data"]
+
+
+def add_followup_request_using_frontend_and_verify(
+    driver, super_admin_user, public_source, super_admin_token, public_group
```

Comment:
Can you merge the LT and SEDM tests into a single file? And could you add a test that submits both an LT and and an SEDM request in a single test?